services/QuillLMS/app/services/learn_worlds_integration/suspended_users_request.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module LearnWorldsIntegration
+  class SuspendedUsersRequest < Request
+    MAX_ITEMS_PER_PAGE = 200
+
+    def endpoint = "#{USER_TAGS_ENDPOINT}?status=suspended&items_per_page=#{MAX_ITEMS_PER_PAGE}"
+
+    def run = fetch_ids_to_suspend
+
+    def fetch_page(page_number)
+      response = HTTParty.get("#{endpoint}&page=#{page_number}", headers:)
+      raise UnexpectedApiResponse unless response&.response&.message == "OK"
```

Comment:
This is intended to bubble up to SyncOrchestrator, in order to catch rate limit issues before the child jobs are enqueued.